src/components/Modal/index.js

```diff
@@ -12,43 +12,42 @@ const MODAL_ANIMATION_DURATION = getCSSMilisecsValue(
 
 const Wrapper = ({ children }) => wrapWithClass(style.inner, children)
 
-const Modal = () => {
-  const {
-    translate,
-    isFullScreen,
-    containerId,
-    containerEl,
-    shouldCloseOnOverlayClick,
-  } = this.props
```

Comment:
Seems that it's my bad from #1157: https://github.com/onfido/onfido-sdk-ui/pull/1157/files#diff-b17c47ef8137161ae8a178243850f1c4R22. So funny that we all didn't catch it 😅

Reply:
Yeah, I was looking at it thinking "I can't believe I missed this" 🙈